vue/configurator/src/components/lib/WidgetProps.ts

```diff
@@ -9,6 +9,11 @@ interface WidgetPropOptions {
   label: string;
 }
 
+interface WidgetThemeDefinition {
+  value: string;
+  options: WidgetPropOptions[];
+}
+
```

Comment:
this works -- was thinking since we have set values for the `theme` stuff (the scheme utility classes), we can be more explicit about what a theme is.